### PR TITLE
fix: invalid ParentObj in output

### LINKED ISSUE #1067: [Bug]:  invalid parent object in output list
### Checklist

- [X] I've searched for similar issues and couldn't find anything matching
- [X] I've included steps to reproduce the behavior

### Affected Components

- [X] K8sGPT (CLI)
- [ ] K8sGPT Operator

### K8sGPT Version

_No response_

### Kubernetes Version

_No response_

### Host OS and its Version

_No response_

### Steps to reproduce

`k8sgpt analyze`

### Expected behaviour

```
1 default/tt8(tt8)
- Error: Service has no endpoints, expected label k=v
```

for those objects without parentObject, the blanket will show nothing as below

```
1 default/tt8()
- Error: Service has no endpoints, expected label k=v
```

### Actual behaviour

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by panpan0000**
The `GetParent()` always return `false` and the invoker never respect its return value.

So add the boolean return value (I believe it means "found" )

Closes #1067

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information

Previous
image
Now
image

### CODE REVIEW

**Comment by arbreezy** [comment]
Nice catch @panpan0000

File: pkg/util/util.go [hunk 1]
```diff
@@ -114,11 +114,11 @@ func GetParent(client *kubernetes.Client, meta metav1.ObjectMeta) (string, bool)
 				if vw.OwnerReferences != nil {
 					return GetParent(client, vw.ObjectMeta)
 				}
-				return "ValidatingWebhook/" + vw.Name, false
+				return "ValidatingWebhook/" + vw.Name, true
 			}
 		}
 	}
-	return meta.Name, false
```
Comment (panpan0000) [inline pkg/util/util.go:121]: I'm not sure about this line . can you help

Comment (panpan0000) [reply pkg/util/util.go:121]: @AlexsJones  , can you help ?

Comment (arbreezy) [reply pkg/util/util.go:121]: sorry @panpan0000 just saw it,
yes, I think an empty string makes more sense, the meta.Name doesn't give any more information in the output
 e.g
`return "", false`

**Review by arbreezy — CHANGES_REQUESTED** [review]

File: pkg/util/util.go (hunk 1 above)
Comment (AlexsJones) [reply pkg/util/util.go:121]: This should return false

Comment (panpan0000) [reply pkg/util/util.go:112]: thank you both, It's fixed.

**Review by arbreezy — APPROVED** [review]

**Comment by arbreezy** [comment]
@panpan0000 we need to fix this test as well , https://github.com/k8sgpt-ai/k8sgpt/blob/main/pkg/util/util_test.go#L85

**Comment by panpan0000** [comment]
> @panpan0000 we need to fix this test as well , https://github.com/k8sgpt-ai/k8sgpt/blob/main/pkg/util/util_test.go#L85

Thank you for reminder. @arbreezy , fixed as well.

**Pull request merged**